sql/core/src/test/scala/org/apache/spark/sql/DataFrameAggregateSuite.scala

```diff
@@ -957,17 +957,4 @@ class DataFrameAggregateSuite extends QueryTest
       assert(error.message.contains("function count_if requires boolean type"))
     }
   }
-
-  test("calendar interval agg support hash aggregate") {
```

Comment:
Hi, @cloud-fan .
Although this is a correct removal, this is a part of `[SPARK-30047][SQL] Support interval types in UnsafeRow`.

Reply:
support interval in `UnsafeRow` is fine. It has perf benefits, not just for hash aggregate, so we shouldn't revert it.

Reply:
Yes, I'm not asking to revert SPARK-30047. I'm saying that it would be great if this PR title and description mentioned this clearly.

Reply:
Added to the PR description.